### PR TITLE
Fix dependencies and add checks to help keep recipe in sync with upstream repo

### LINKED ISSUE #9: Incorrect dependencies
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

The dependencies of this package do not match those of the [upstream repo](https://github.com/iwpnd/flashgeotext/blob/v0.5.3/pyproject.toml).

### Installed packages

```shell
N/A
```

### Environment info

```shell
N/A
```

### PR DESCRIPTION
**Pull request by DManowitz**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes #9

### CODE REVIEW

**Comment by DManowitz** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+bot:
+  inspection: update-grayskull
```
Comment (iwpnd) [inline conda-forge.yml]: What is that?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,27 +10,31 @@ source:
...
 test:
   imports:
     - flashgeotext
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://flashgeotext.iwpnd.pw
```
Comment (iwpnd) [inline recipe/meta.yaml]: can you change that to the GitHub readme while you're at it?

**Review by iwpnd — APPROVED** [review]

**Comment by iwpnd** [comment]
Thank you 🙏🏻

File: conda-forge.yml (hunk 1 above)
Comment (DManowitz) [reply conda-forge.yml]: [Grayskull](https://github.com/conda/grayskull) is a conda recipe generator.  I [had been told](https://github.com/conda-forge/conda-forge.github.io/issues/2062#issuecomment-1890753604) that adding this line this was a way to help improve dependency updating for conda-forge recipes.  However, actually, looking at [this](https://conda-forge.org/docs/maintainer/conda_forge_yml/#bot), perhaps I should have set this to `update-all`

File: recipe/meta.yaml (hunk 2 above)
Comment (DManowitz) [reply recipe/meta.yaml]: The `about`?

Comment (iwpnd) [reply recipe/meta.yaml]: yeah, those links to iwpnd.pw are no longer valid. Could just link to readme instead.

**Comment by DManowitz** [comment]
@iwpnd Updated my commits.

**Pull request merged**